go.mod

```diff
@@ -1 +1,5 @@
 module github.com/alexandrevicenzi/go-sse
+
+go 1.12
+
+require github.com/go-chi/chi v4.0.2+incompatible
```

Comment:
This should not be here as it's not a dependency. Maybe examples needs to be renamed to _examples.

Reply:
Renamed examples to _examples.

Reply:
I think you should consider cleaning test cache on travis before running them.